Sources/NostrSDK/Events/TextNoteRepostEvent.swift

```diff
@@ -12,6 +12,10 @@ import Foundation
 /// See [NIP-18](https://github.com/nostr-protocol/nips/blob/master/18.md#reposts).
 public final class TextNoteRepostEvent: GenericRepostEvent {
     
+    override class var kind: EventKind {
+        .repost
```

Comment:
Curious about this, it looks to me like it's overriding the value from `GenericRepostEvent` but they are the same

Reply:
Dang, you're right! Good catch. Fixed and added a test that would've caught it.

Reply:
Nice! 👍